libri-whitepaper.tex

```diff
@@ -29,19 +29,19 @@
 \author{Drausin Wulsin}
 \maketitle
 
 \begin{abstract}
-We describe Libri, a decentralized storage network intended for heterogenous healthcare data. This focus allows Libri to be narrow in scope and opinionated in its API, which enforces end-to-end encryption and secure sharing. Libri does not depend on any blockchain or distributed consensus protocol, simplifying its implementation and allowing its performance to scale very well with horizontal cluster size. Libri documents are automatically (re-)replicated across network peers to ensure robustness to planned and unplanned peer outages. The Libri API exposes simple \Put{}/\Get{} endpoints and a streaming \ttt{Subscribe} endpoint for storage notifications across the entire network. We describe initial experiments measuring request load and cluster size on request latency, rate, and throughput. In particular, a modest 64-peer cluster with 1 CPU and 6 GB RAM per peer easily handles over a million documents uploaded per day with median \Get{} and \Put{} latencies of 6 ms and 22 ms, respectively. Request latency and throughput scale well with horizontal cluster size increasing from 8 to 64 peers. These initial results give us confidence that Libri can scale to efficiently handle production-level load across all US healthcare organizations. We hope to build a community from these public and private healthcare organizations to run peers in the Libri network, which will provide a shared, durable, and accessible repository for short- and long-term health data. 
+We describe Libri, a decentralized storage network suitable for cooperative enterprise data consortiums. Although Libri is suitable for a range of enterprise domains, its security and sharing capabilities are particularly well-suited to health data storage and sharing. Libri does not depend on any blockchain or distributed consensus protocol, simplifying its implementation and allowing its performance to scale very well with horizontal cluster size. Libri documents are automatically (re-)replicated across network peers to ensure robustness to planned and unplanned peer outages. The Libri API exposes simple \Put{}/\Get{} endpoints and a streaming \ttt{Subscribe} endpoint for storage notifications across the entire network. We describe initial experiments measuring request load and cluster size on request latency, rate, and throughput. In particular, a modest 64-peer cluster with 1 CPU and 6 GB RAM per peer easily handles over a million documents uploaded per day with median \Get{} and \Put{} latencies of 6 ms and 22 ms, respectively. Request latency and throughput scale well with horizontal cluster size increasing from 8 to 64 peers. These initial results give us confidence that Libri can scale to efficiently handle production-level load across many common enterprise use-cases, such as health data sharing for large US provider organizations.
```

Comment:
`suitable for cooperative enterprise data consortiums` -> `intended for heterogenous healthcare data. While its initial focus is healthcare, its performance, security, and sharing capabilities make it well-suited for data consortiums across a number of industries.`